src/Common/src/System/Security/Cryptography/KeyBlobHelpers.cs

```diff
@@ -10,6 +10,45 @@ namespace System.Security.Cryptography
 {
     internal static class KeyBlobHelpers
     {
+        internal static byte[] ToUnsignedIntegerBytes(this ReadOnlyMemory<byte> memory, int length)
```

Comment:
where are those actually used? I can't see any references neither in this PR nor in the tree

Reply:
Ahh I think your [comment above](https://github.com/dotnet/corefx/pull/41376#issuecomment-535728377) just explained it

Reply:
https://github.com/dotnet/corefx/blob/fae96a267944e45c9b6a90f4065ba4e1c803c93a/src/Common/src/System/Security/Cryptography/EccKeyFormatHelper.cs#L403-L421, which for some reason GitHub says has no changes.

Reply:
Interestingly, looking at the commit in my fork shows the diff...

https://github.com/bartonjs/corefx/commit/fae96a267944e45c9b6a90f4065ba4e1c803c93a#diff-008636d0c2c61ae86f91d3d955443f25

Reply:
I've put some comments directly on the commit